apps/src/applab/applab.js

```diff
@@ -1306,6 +1309,9 @@ Applab.beginVisualizationRun = function() {
   Applab.running = true;
   $('#headers').addClass('dimmed');
   $('#codeWorkspace').addClass('dimmed');
+  if (!Applab.isReadOnlyView) {
+    $('.droplet-main-canvas').css('background-color', '#E5E5E5');
```

Comment:
Since these colors are going to be used in lots of places, consider adding them to our `shared/css/color.scss` file and referencing it from `@cdo/apps/util/color`. (Looks like white is already there)